androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -269,6 +273,7 @@ private void launchWhenSplashScreenReady(TrustedWebActivityIntentBuilder builder
         }
         Log.d(TAG, "Launching Trusted Web Activity.");
         TrustedWebActivityIntent intent = builder.build(mSession);
+        intent.getIntent().putExtra(EXTRA_STARTUP_UPTIME_MILLIS, SystemClock.uptimeMillis());
```

Comment:
I think we should start measuring the timestamp when the activity starts, not when we launch the intent (this would otherwise fail to measure the animation delay); so we should try to pass a timestamp from the Activity. taking a timestamp in launch() might be also fine if it's close enough.